src/main/java/org/openhab/binding/zwave/handler/ZWaveControllerHandler.java

```diff
@@ -252,26 +252,22 @@ public void handleConfigurationUpdate(Map<String, Object> configurationParameter
                     continue;
                 }
 
-                if (cfg[1].equals("softreset") && value instanceof BigDecimal
-                        && ((BigDecimal) value).intValue() == ZWaveBindingConstants.ACTION_CHECK_VALUE) {
+                if (cfg[1].equals("softreset") && value instanceof Boolean && ((Boolean) value) == true) {
                     controller.requestSoftReset();
 
-                    value = new BigDecimal(0);
-                } else if (cfg[1].equals("hardreset") && value instanceof BigDecimal
-                        && ((BigDecimal) value).intValue() == ZWaveBindingConstants.ACTION_CHECK_VALUE) {
+                    value = false;
+                } else if (cfg[1].equals("hardreset") && value instanceof Boolean && ((Boolean) value) == true) {
                     controller.requestHardReset();
 
-                    value = new BigDecimal(0);
-                } else if (cfg[1].equals("exclude") && value instanceof BigDecimal
-                        && ((BigDecimal) value).intValue() == ZWaveBindingConstants.ACTION_CHECK_VALUE) {
+                    value = false;
+                } else if (cfg[1].equals("exclude") && value instanceof Boolean && ((Boolean) value) == true) {
                     controller.requestRemoveNodesStart();
 
                     value = new BigDecimal(0);
```

Comment:
I would not say I'm familiar with your code, but reviewing your commits helps me to learn it a little. In looking through this commit, this line stood out because of its asymmetry with the other changes. Should this line be replaced with `value = false`?

Reply:
Good spot - thanks. Missed this one...